ohw22/seattle/index.md

```diff
@@ -31,15 +31,49 @@ We welcome participants from all career stages and encourage applications from g
 3. We encourage students to communicate with the global virtual event and other satellite events to augment their OHW experience.
 
 
-**Logistics**
+### Location
+All activities will be in Ocean Sciences Building (OSB)  https://www.washington.edu/maps/#!/ocn on the University of Washington Seattle campus.
+- OSB 425 will be main location for tutorials and large group discussions
+- Other OSB locations can be leveraged for breakout discussions
```

Comment:
This section (ln 34-37) could be moved up to the location section a few lines up (ln 15), and it looks like the link to the UW map didn't hyperlink properly.

Reply:
Good catch! That sounds good. Or we could just delete the current, higher-level Location section. Either way is fine with me.

Reply:
Ok, I moved the location detail to the top level so that people can easily see it.